.pre-commit-config.yaml

```diff
@@ -7,8 +7,22 @@ repos:
     hooks:
     -   id: check-case-conflict
     -   id: check-json
-    -   id: check-yaml
     -   id: check-merge-conflict
+    -   id: check-yaml
     -   id: fix-byte-order-marker
+    -   id: check-added-large-files
+        args: ['--maxkb=100']
+    -   id: mixed-line-ending
+
+-   repo: https://github.com/lorenzwalthert/precommit
```

Comment:
any idea where this comes from and who lorenz walthert is?

Reply:
haha no idea at all..

Reply:
maybe @pfuehrlich-pik or @giannou ?